lib/modules/datasource/maven/readme.md

```diff
@@ -0,0 +1,16 @@
+### Fetching changelogs from API
+
+In case your repository doesn't hold a `CHANGELOG.md` file and you need to fetch changelogs from an API (just like the one from [Gitlab](https://docs.gitlab.com/ee/api/releases) or [Github](https://docs.github.com/en/rest/releases)), you need to configure `Maven` so that `renovatebot` is able to determine the `sourceUrl` when fetching info about the artifact.
```

Comment:
This sentence is too long. It needs a rewrite so it:

- Explains better _why_ one may need to fetch changelogs from a API. I think Renovate can usually find changelogs on its own. So what makes this different?
- Explains if this is aimed at end-users of Renovate, or at maintainers of repositories that Renovate can fetch changelogs from?

Reply:
Ok. I will rewrite to be more clear.

Is true, Renovate can find changelogs automatically. It may do so from the `CHANGELOG.md` file, but it can also fetch them from the API alternatively. For it to work, it requires the `sourceUrl` to be resolved and that can only happen if it is defined in the artifact's pom. In fact, I'm thinking that may be the case on both situations, so maybe it doesn't make sense to specifically refer the API case.
This is aimed at the person publishing the artifact, so that the end-users can see changelogs on the PRs. In my specific case I am both (publisher and end-user), so I was biased when writing this. I will make that also more clear.

Thanks

Reply:
Published a new version. Please have a look 🙂